### PR TITLE
fix cosmos header

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: dcos/cosmospackage.py [hunk 1]
```diff
@@ -59,3 +59,13 @@ def _get_cosmos_header(request_name):
 
     return {"Accept": _get_header("{}-response".format(request_name)),
             "Content-Type": _get_header("{}-request".format(request_name))}
+
+
+def _get_capabilities_header():
+    """Returns header fields needed for a valid request to cosmos capabilities
+    endpoint
```
Comment (jsancio) [inline dcos/cosmospackage.py]: add line.

File: dcos/cosmospackage.py [hunk 2]
```diff
@@ -59,3 +59,13 @@ def _get_cosmos_header(request_name):
...
+    :returns: header information
+    :rtype: str
```
Comment (jsancio) [inline dcos/cosmospackage.py]: `rtype: dict`

File: dcos/cosmospackage.py [hunk 3]
```diff
@@ -59,3 +59,13 @@ def _get_cosmos_header(request_name):
...
+    """
+    header = "application/vnd.dcos.capabilities+json;charset=utf-8;version=v1"
+    return {"Accept": header, "Content-Type": header}
```
Comment (jsancio) [inline dcos/cosmospackage.py]: The body is empty. We should leave out `Content-Type`.

**Comment by tamarrow** [comment]
@jsancio updated

**Comment by tamarrow** [comment]
#489

File: dcos/cosmospackage.py [hunk 4]
```diff
@@ -59,3 +59,14 @@ def _get_cosmos_header(request_name):
 
     return {"Accept": _get_header("{}-response".format(request_name)),
             "Content-Type": _get_header("{}-request".format(request_name))}
+
+
+def _get_capabilities_header():
+    """Returns header fields needed for a valid request to cosmos capabilities
+    endpoint
```
Comment (jsancio) [inline dcos/cosmospackage.py]: We need to add a new line between the sentence above and the params below.

**Comment by jsancio** [comment]
LGTM!

**Pull request merged**